.github/workflows/kubeapps.yaml

```diff
@@ -487,9 +500,8 @@ jobs:
         with:
           name: e2e_tests_${{ matrix.tests_group }}_reports
           path: integration/reports
-      - name: Setup upterm session
-        uses: lhotari/action-upterm@v1
-        if: failure() && env.DEBUG_MODE == 'true'
+      - uses: lhotari/action-upterm@v1
+        if: false
```

Comment:
Leftover maybe?

Reply:
yeah, leftover a reminder for when I need it 😅 But yeah, it would be better removed